src/components/calcite-card/calcite-card.tsx

```diff
@@ -0,0 +1,186 @@
+
+  /**
+   * Indicates whether the card is selected. Toggles when a card is clicked.
+   */
+  @Prop({ reflect: true }) selected = false;
+
+  /**
+   * The theme of the card.
+   */
+  @Prop({ reflect: true }) theme: "light" | "dark" = "light";
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteCardElement;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  // --------------------------------------------------------------------------
+
+  connectedCallback() {
+    //prop validations
+      let scales = ["s", "m", "l"];
+      if (!scales.includes(this.imageScale)) this.imageScale = "m";
+
+      let themes = ["dark", "light"];
+      if (!themes.includes(this.theme)) this.theme = "light";
+  }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Public Methods
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Used to toggle the selection state.
+   */
+  @Method() async toggleSelected(coerce?: boolean) {
+    if (this.disabled) {
+      return;
+    }
+    this.selected = typeof coerce === "boolean" ? coerce : !this.selected;
+  }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  cardClickHandler = (event: MouseEvent): void => {
+    event.preventDefault();
+    if (this.disabled) {
+      return;
+    }
+    this.toggleSelected();
+  };
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+
+  renderHeader(): VNode {
+    const hasHeader = this.el.querySelector(`[slot=${SLOTS.header}]`);
+
+    return hasHeader ? (
+      <header class={CSS.header}>
+        <slot name={SLOTS.header} />
+      </header>
+    ) : null;
+  }
+
+  renderThumbnail(): VNode {
+    const hasThumbnail = this.el.querySelector(`[slot=${SLOTS.thumbnail}]`);
+
+    return hasThumbnail ? (
+      <div class={CSS.thumbnail}>
+        <slot name={SLOTS.thumbnail} />
+      </div>
+    ) : null;
+  }
+
+  renderFooter(): VNode {
+    const leadingFooter = this.el.querySelector(
+      `[slot=${SLOTS.footerLeading}]`
+    );
+    const trailingFooter = this.el.querySelector(
+      `[slot=${SLOTS.footerTrailing}]`
+    );
+
+    const hasFooter = leadingFooter || trailingFooter;
+
+    return hasFooter ? (
+      <footer class={CSS.footer}>
+        <slot name={SLOTS.footerLeading} />
+        <slot name={SLOTS.footerTrailing} />
+      </footer>
+    ) : null;
+  }
+
+  renderScrim(): VNode {
```

Comment:
question for @asangma: do we even need a scrim for this component? Does a card need a loading or disabled state?

Reply:
I think loading state makes sense, probably don't need a disabled state since "loading" state prevents interaction and can be used to cover up anything loading-in. Thoughts @asangma ?